vscode/package.json

```diff
@@ -456,14 +456,14 @@
           "type": "string",
           "default": "",
           "description": "Path to the analyzer binary. If not set, the extension will use the bundled binary.",
-          "scope": "machine",
+          "scope": "resource",
           "order": 2
         },
         "konveyor.kaiRpcServerPath": {
           "type": "string",
           "default": "",
           "description": "Path to the rpc-server binary. If not set, the extension will use the bundled binary.",
-          "scope": "machine",
+          "scope": "resource",
```

Comment:
Setting to resource allows writing to Workspace settings instead of limiting to user settings ( which was failing before this change).